lib/hanami/helpers/html_helper/empty_html_node.rb

```diff
@@ -124,7 +124,11 @@ def boolean_attribute(attribute_name, _value)
         end
 
         def attribute(attribute_name, value)
-          %(#{ATTRIBUTES_SEPARATOR}#{attribute_name}="#{value}")
+          if attribute_name.to_sym == :value
+            %(#{ATTRIBUTES_SEPARATOR}#{attribute_name}="#{Hanami::Utils::Escape.html(value)}")
+          else
+            %(#{ATTRIBUTES_SEPARATOR}#{attribute_name}="#{value}")
```

Comment:
Probably yes.

Please also consider to use `html_attribute` instead https://github.com/hanami/utils/blob/ca91c336ffebb43f1632ffd398b4281b3b7315a5/lib/hanami/utils/escape.rb#L453